scripts/build_deps.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
+
+# Build dependencies
+#
+#   build_deps.sh <build-csdk>
+#
+#   Options:
+#   build-csdk: 1 to build EdgeX C SDK, 0 to skip
+#
+# Assumes WORKDIR is /device-coap
```

Comment:
Better to define this WORKDIR information in README.md. It took a while for me to find it and correct myself.

Reply:
Thanks for the experience report. Addressed in b517d06.